api-admin/src/main/java/bio/terra/pearl/api/admin/service/portal/PortalExtService.java

```diff
@@ -56,10 +55,6 @@ public PortalEnvironmentConfig updateConfig(
       EnvironmentName envName,
       PortalEnvironmentConfig newConfig,
       AdminUser user) {
-    if (!user.isSuperuser()) {
-      throw new PermissionDeniedException(
-          "You do not have permissions to update portal configurations");
-    }
```

Comment:
this restriction was added before out of an abundance of caution.  But now we need to allow this so that HeartHive can make their site live/password protected on the day of their conference without our intervention.